src/cloud-element-templates/properties-panel/properties/CustomProperties/BooleanProperty.js

```diff
@@ -0,0 +1,34 @@
+import { useService } from 'bpmn-js-properties-panel';
```

Comment:
I'm wondering if we want to switch to `custom-properties` for our naming as we moved things around. We don't usually camel-case folder names.

Reply:
I'll adjust it

Reply:
Done in [0105490](https://github.com/bpmn-io/bpmn-js-element-templates/pull/64/commits/0105490766757e5d1e65652c38528a5186b80d10)